usr/lib/tpm_stdll/tpm_specific.c

```diff
@@ -3513,6 +3734,63 @@ CK_RV token_specific_aes_ecb(STDLL_TokData_t * tokdata,
     *out_data_len = in_data_len;
 
     return CKR_OK;
+#else
+    CK_RV rc;
+    int outlen;
+    unsigned char akey[32];
```

Comment:
do we have a constant for AES_256_KEY_SIZE ?

Reply:
yes we do. done.